idaes/power_generation/carbon_capture/compression_system/tests/test_compressor.py

```diff
@@ -52,11 +52,12 @@ def build_unit():
     # compressor first stage
     p = 1.13937 * 1e5  # Pa
     t = 40.0113 + 273.15  # K
-    fin = 1689.31  # mol/s
+    fin = 1548.52  # mol/s 1689.31
```

Comment:
What is the trailing 1689.31 for?

Reply:
It is a wrong value of inlet flowrate. I update the new value for checking the pressure ratio.